scripts/pam_swap.py

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2016 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""
+Script to handle swapping of PAMs.
+"""
```

Comment:
per Slack conversation, this script is fairly temporary to implement a specific change, so doesn't need the same level of testing.